geth/account/adapter.go

```diff
@@ -0,0 +1,57 @@
+package account
+
+import (
+	"crypto/ecdsa"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/status-im/status-go/extkeys"
+	"github.com/status-im/status-go/geth/common"
+)
+
+// accountNode narrows responsibility geth/common.NodeManager interface
+type accountNode interface {
+	// AccountKeyStore returns narrowed responsibility of go-ethereum/accounts/keystore.KeyStore
+	AccountKeyStore() (accountKeyStorer, error)
+	// AccountManager returns narrowed responsibility of go-ethereum/accounts.Manager
+	AccountManager() (gethAccountManager, error)
+	// WhisperService returns narrowed responsibility  go-ethereum/whisper/whisperv5.Whisper
+	WhisperService() (whisperService, error)
+}
+
+//gethAccountManager narrows responsibility of go-ethereum/accounts.Manager
```

Comment:
Thanks for comments, only a little formal wish: Beginning space and a period in the end.

```
// I'm a comment.
```

Reply:
@themue Why not. Just added..